content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -38,18 +42,59 @@ and
-Without using the BOM, you would need to go and update the dependencies in your plugin to match those required by Jenkins, and then keep changing these as your `jenkins.version` changes.
-With the BOM all you have to do is import the link:https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml[jenkins-bom] for the version of Jenkins you are building against, and the versions of these and other common dependencies will be matched to that version of Jenkins.
+The link:https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[Apache Maven Bill Of Materials (BOM)]
+defines the versions of various libraries used by Jenkins.
+It is included in version 2.195 (and LTS version 2.164.x) and later of Jenkins Core.
 
-To use the Jenkins Core BOM in your plugin, just use plugin-pom (4.0 or later).
+If you use Maven to build your plugin,
+dependencies are defined in the `dependencyManagement` section of you _pom.xml_ file.
+You can import the appropriate Jenkins BOM to your _pom.xml_ file
+so that dependency versions are automatically synchronised
+with the version of Jenkins against which you are building.
 
-== Jenkins Plugin BOM
+To use the Jenkins Core BOM in your plugin:
+
+////
+Does this need a hint about how to be sure one is using this?
+////
+. Be sure that you are using
+link:https://github.com/jenkinsci/plugin-pom/[Parent POM for Jenkins Plugins] 
+(4.0 or later).
+. Go to the the link:https://github.com/jenkinsci/bom[jenkinsci/bom] repository.
+. Go to the `bom-*` folder that corresponds to
+the Jenkins version against which you are building.
+. Open the _pom.xml_ file in that folder.
+. Copy the `dependencyManagement` section from the _pom.xml_ file.
+. Paste the `dependencyManagement` section into the _pom.xml_ file for your plugin.
+If your file already has a `dependencyManagemnt` section,
+replace it with this text.
+Your _pom.xml_ file should only have one `dependencyManagement` section.
+. Rebuild your plugin to see if this solves your dependency issues.
+
+The _README_ file in the jenkinsci/bom repository includes background
+and more details about using the BOM in your plugin.
 
-See the link:https://github.com/jenkinsci/bom[jenkinsci/bom] repository
 
 == Dependabot
 
+////
+PLEASE CHECK THIS CAREFULLY!
+////
+The GitHub link:https://github.com/dependabot[Dependabot] feature
+can be used to simplify dependency management for builds
+no matter what build tools are used.
 See the link:https://groups.google.com/forum/#!msg/jenkinsci-dev/XMllKuWLO_8/H7nN70D1AwAJ[discussion here].
 
+link:https://github.com/jenkinsci/archetypes/blob/master/.github/dependabot.yml[dependabot.yml]
+is the default in the template/archtype.
+
+Some information about how to implement this and some know obstacles
+are in link:https://docs.google.com/document/d/1hRrH8PSCswBQgY_Q-7eHCHCVZHJOl4XgQQCswdUmpKY/edit#heading=h.oqikyovwtohu[Dependabot Evaluation Notes].
```

Comment:
these discussions are very dated almost 3 years old now.

Dependabot is a well established github feature, we should probably just link to the official docs: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/enabling-and-disabling-dependabot-version-updates

and to our default template:

```suggestion
link:https://github.com/jenkinsci/archetypes/blob/master/.github/dependabot.yml[dependabot.yml]
is the default in the template/archtype.
```